### PR TITLE
Created a simple rest api.

### PR DESCRIPTION
**Pull request by RayanZaki**
I have submitted a simple rest-api using fast api, tested the predicive models with both datasets. you can find attached the result of my work. There is still off course some better exception handling and refactoring to do. but for now It looks suitable. please inform me of any feedback.
image
image

### CODE REVIEW

File: entire_model.pth
Comment (1hachem) [inline entire_model.pth:1]: what is this doing here ?

File: config.py
Comment (1hachem) [inline config.py:1]: what is this file about ?

Comment (RayanZaki) [reply config.py:1]: Here I saved the training data mean and standard deviation that was used to normalize the data before being passed to the model. In order to normalize the new input data with the same scale as the training data, I stored them in this file to use them in the api level

File: entire_model.pth
Comment (RayanZaki) [reply entire_model.pth:1]: I have exported the model to this file in order to decouple the training process from the inference. Here for the predictions we will use the exported model. If you are talking about the file structure, I figured it was a good place to put it in order to be easily accessible from the api and the training files

Comment (1hachem) [reply entire_model.pth:1]: it would be better to put it inside models and to re-name it

**Comment by 1hachem** [comment]
hey ! there is a conflict try to reslove it 🔥

File: config.py
Comment (1hachem) [reply config.py:1]: moved this inside `data/`, what do you think ?

Comment (RayanZaki) [reply config.py:1]: that seems more suitable

File: entire_model.pth
Comment (RayanZaki) [reply entire_model.pth:1]: Thank you so much for the feedback, I will take it into consideration. It is much appreciated

**Comment by RayanZaki** [comment]
> hey ! there is a conflict try to reslove it 🔥

Ok, I will do it as soon as I am available, currently I am afk

**Comment by RayanZaki** [comment]
> > hey ! there is a conflict try to reslove it 🔥
> 
> Ok, I will do it as soon as I am available, currently I am afk

done

**Review by 1hachem — APPROVED** [review]

**Comment by 1hachem** [comment]
we will contact you soon to setup an interview 😄

**Comment by RayanZaki** [comment]
> we will contact you soon to setup an interview 😄 

That would be great, thank you so much for your time.

**Pull request merged**